core/management/commands/users_and_groups.py

```diff
@@ -44,6 +44,7 @@
     "add_behavioralhealthnotes",
     "change_behavioralhealthnotes",
     "view_behavioralhealthnotes",
+    "view_sites",
```

Comment:
This should be 'site' singular, to match the new model name.

Reply:
Gotcha, so these are only ever going to be displaying a single site? I was trying to make it match whether you could be viewing one or more sites, but I'll just make it match the model.

Reply:
Permissions are not that fine grained out the box. There are four available ones, which are verb/CRUD based. 
https://docs.djangoproject.com/en/3.1/topics/auth/default/#default-permissions

Django REST Frameworks does not permission GET requests, like MVC Django does, so in `core/permissions.py`, we modified the class slightly to enable this feature.